app/views/organizations/_form.html.erb

```diff
@@ -1,37 +1,38 @@
 <%= form_with(model: organization, local: true) do |form| %>
   <% if organization.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(organization.errors.count, "error") %> prohibited this organization from being saved:</h2>
-
-      <ul>
+    <div class="bg-washed-red fw6 br3 pa3 mb4" role="alert">
+      <div class="fw7 mb2"><%= pluralize(organization.errors.count, "error") %> prohibited this organization from being saved:</div>
+      <ul class="ml4">
         <% organization.errors.full_messages.each do |message| %>
           <li><%= message %></li>
         <% end %>
       </ul>
     </div>
   <% end %>
 
-  <div class="field">
-    <%= form.label :name %>
-    <%= form.text_field :name %>
-  </div>
-
-  <div class="field">
-    <%= form.label :contact %>
-    <%= form.text_field :contact %>
-  </div>
-
-  <div class="field">
-    <%= form.label :email %>
-    <%= form.text_field :email %>
+  <div class="cf mb3">
+    <div class="w-50-l">
+      <div class="mb3">
+        <%= form.label :name, class: "db mb1 fw6" %>
+        <%= form.text_field :name, class: "pa3 br2 b--black-20 w-100" %>
+      </div>
+      <div class="mb3">
+        <%= form.label :contact, class: "db mb1 fw6" %>
+        <%= form.text_field :contact, class: "pa3 br2 b--black-20 w-100" %>
+      </div>
+      <div class="mb3">
+        <%= form.label :email, class: "db mb1 fw6" %>
+        <%= form.text_field :email, class: "pa3 br2 b--black-20 w-100" %>
+      </div>
+      <div class="mb3">
+        <%= form.label :phone, class: "db mb1 fw6" %>
+        <%= form.text_field :phone, class: "pa3 br2 b--black-20 w-100" %>
+      </div>
+    </div>
   </div>
 
-  <div class="field">
-    <%= form.label :phone %>
-    <%= form.text_field :phone %>
-  </div>
+  <%= form.button class: "bg-blue white bg-animate hover-bg-dark-blue fw6 br2 pv3 ph4 mr4 mb3" %>
+  <%= link_to 'Cancel Update', @organization, class: "link underline hover-blue dib mr4" %>
+  <%= link_to 'Delete', organization, class: "link underline hover-red dib", method: :delete, data: { confirm: 'Are you sure?' } %>
```

Comment:
This form _form partial will render when creating new organizations too. Perhaps we can add a conditional here to only show the Delete link if the organization is already created. What do you think?

```ruby
<% if organization.persisted? %>
  <%= link_to 'Delete', organization, class: "link underline hover-red dib", method: :delete, data: { confirm: 'Are you sure?' } %>
<% end %>
```

Reply:
My thoughts exactly. I figured that could be addressed when the Admin view is addressed but if it's available I'll add it, thanks!